descriptors/ModuleDescriptor-template.json

```diff
@@ -4,35 +4,24 @@
   "provides": [
     {
       "id": "rtac",
-      "version": "1.4",
+      "version": "2.0",
       "handlers": [
         {
-          "methods": ["GET"],
-          "pathPattern": "/rtac",
-          "permissionsRequired": ["rtac.holdings.collection.get"]
-        },
-        {
-          "methods": ["GET"],
-          "pathPattern": "/rtac/{id}",
-          "permissionsRequired": ["rtac.holdings.item.get"],
+          "methods": ["POST"],
+          "pathPattern": "/rtac/batch",
+          "permissionsRequired": ["rtac.batch.post"],
```

Comment:
Where is the `rtac.batch.post` permission defined? I see it as a sub permission of `rtac.all`, but I don't see the permission actually being created like the other RTAC endpoint permissions.

Reply:
Fixed